### DISCUSSION TITLE
This is amazing!

### DISCUSSION THREAD

**Discussion by sebasptsch** [fjodor-rybakov/discord-nestjs#795] [General]
Just though I would say that this integration is amazing! Everything follows NestJS plus with the added awesomeness of a discord bot framework that can work in conjunction with an API. I love it. Thanks for all your hard work!

**Comment by fjodor-rybakov** [comment]
Thank you! 🙃

**Comment by wh1337** [comment]
I literally came here to say the exact same thing! This project is truly amazing.

**Comment by rconjoe** [comment]
Yep, I had to come by and mention how cool it is that you took the time to make this @fjodor-rybakov. One of my favorite projects I have found in a long time. :100: :1st_place_medal:
